examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md

```diff
@@ -1,44 +1,66 @@
-# Account IAM roles creation example
+# Account-Wide IAM Roles
 
-As a prerequisite for ROSA STS clusters, 
-this example demonstrates the process of creating account IAM roles.
-To run it:
+Prior to creating a ROSA STS cluster, you must create the required account-wide roles and policies. For more information, see [Account-wide IAM role and policy reference](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-sts-about-iam-resources#rosa-sts-account-wide-roles-and-policies_rosa-sts-about-iam-resources) in the Red Hat Customer Portal.
 
-* Decide STS account_role_prefix    
-    ```
-    export TF_VAR_account_role_prefix=...
-    ```
+## Prerequisites
 
-* Provide STS openshift_version
-    ```
-    export TF_VAR_openshift_version=...
+* You have an AWS account.
+* You have installed the latest version ROSA CLI (`rosa`)
+* You must have an offline OCM token. This token can be generated in the [Red Hat Hybrid Cloud Console](https://console.redhat.com/openshift/token).
+* You have installed Terraform. See the [Hashicorp Terraform page](https://developer.hashicorp.com/terraform/downloads) for the latest version.
+
+## Account-wide IAM role creation
+
+1. To run the `terraform apply` you need to set up various variables. This guide uses environmental variables. For a more expansive explanation of Terraform variables, see [Managing Variables](https://developer.hashicorp.com/terraform/enterprise/workspaces/variables/managing-variables) in the Terraform documentation.
```

Comment:
I don't love "various variables." While I applaud the alliteration, do we need "various"? You could also ammend the third sentence to just "For more on Terraform variables, see..."